Remotion/Web/Core/UI/Controls/FormGridManager.cs

```diff
@@ -1675,13 +1675,13 @@ private void RegisterValidationErrors (FormGridRow dataRow)
         {
           ValidationError validationError = (ValidationError) validationErrorList[i]!;
           //  Get validation message
-          string validationMessage = validationError.ValidationMessage;
+          PlainTextString validationMessage = validationError.ValidationMessage;
           //  Get tool tip, tool tip is validation message
-          if (!string.IsNullOrEmpty (validationMessage))
+          if (!validationMessage.IsEmpty)
           {
             if (toolTip.Length > 0)
               toolTip.AppendLine();
-            toolTip.Append (validationMessage);
+            toolTip.Append (validationMessage.GetValue());
           }
         }
         dataRow.ValidationMarker = CreateValidationMarker (toolTip.ToString());
```

Comment:
Update the api to pass a PlainTextString?

Reply:
What do you mean? Adding a `StringBuilder` method on `PlainTextString` or an extension method?

Reply:
CreateValidationMarker (string) => CreateValidationMarker (PlainTextString)